apollo-backend/src/test/java/io/logz/apollo/SlaveTest.java

```diff
@@ -71,12 +72,31 @@ public void testLastKeepalive() throws ApolloClientException {
         waitUntilSlaveStarted(slaveConfiguration);
 
         Date slaveLastKeepalive = slaveDao.getSlave(slaveId, slaveEnvironment.getId()).getLastKeepalive();
-        Common.waitABit(5);
+        int keepaliveIntervalSeconds = standaloneApollo.getInstance(ApolloConfiguration.class).getSlave().getKeepaliveIntervalSeconds();
```

Comment:
I think this is the wrong standalone, no?
I will work, but if it will start with diff keepalive it can create an issue

Reply:
Why is it the wrong standalone? So what are you suggesting?

Reply:
It's the master one, not the slave - no?

Reply:
Yes, but from the mater's configuration I get the slave's configuration